app/views.py

```diff
@@ -1,3 +1,31 @@
 from django.shortcuts import render
+from allauth.account.views import SignupView
+from app.forms import CustomUserCreationForm
+from django.core.mail import send_mail
+# from django.contrib.auth.views import LoginView
```

Comment:
If you don't need this code, you don't need comment-out it - simply remove it. All changes history in Git!